### PR TITLE
docs(readme): fix broken url workflow icons

### LINKED ISSUE #370: docs(readme): url workflow icons results in 404 not found
## Description

In the README.md file, the URL to the Workflow icons results in a 404 error: page not found.

Screenshot 2019-10-30 at 21 19 23
URL consulted: https://opensource.adobe.com/spectrum-css/icons/

Correct URL: https://spectrum.adobe.com/page/icons/

## Link to documentation

README.md file: https://github.com/adobe/spectrum-css/blob/master/README.md

## Additional context

URL links to the incorrect page.

### PR DESCRIPTION
**Pull request by donroyco**
## Description
This fixes #370 

## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

## Screenshots
Before:
Screenshot 2019-10-30 at 21 19 23

After:
Screenshot 2019-10-30 at 21 26 05

## To-do list

- [x] If my change impacts other components, I have tested to make sure they don't break.
- [x] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] This pull request is ready to merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ These icons are released within in the bundle package as `@spectrum-css/spectrum
 loadIcons('node_modules/@adobe/spectrum-css-workflow-icons/dist/spectrum-icons.svg');
 ```
 
-You can then use the icons in your app. Visit the [Spectrum CSS workflow icon list](http://opensource.adobe.com/spectrum-css/icons/) and click on any icon to get the markup.
+You can then use the icons in your app. Visit the [Spectrum CSS workflow icon list](https://spectrum.adobe.com/page/icons/) and click on any icon to get the markup.
```
Comment (lazd) [inline README.md]: Let's change the link text to be `Spectrum workflow icon list`

Comment (donroyco) [reply README.md]: Let me arrange that!

**Comment by donroyco** [comment]
@lazd Fixed your comment.

**Comment by lazd** [comment]
@donroyco excellent, thanks!

**Pull request merged**